.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature enhancement request
+about: Suggest an idea for this project
+
+---
+
+/kind feature
+
+**Describe the solution you'd like**
```

Comment:
I would keep it shorter like `**Describe your solution:**` or even shorter just `**Description**`

```Markdown
**Description**
[A clear and concise description of what your proposal. What problem does it solve?]
```